packages/vector_graphics_compiler/lib/src/geometry/path.dart

```diff
@@ -453,6 +455,41 @@ class Path {
         other.fillType == fillType;
   }
 
+  /// Compute the bounding box for the given path segment.
+  Rect bounds() {
+    if (_commands.isEmpty) {
+      return Rect.zero;
+    }
+    double smallestX = double.maxFinite;
+    double smallestY = double.maxFinite;
+    double largestX = 0;
+    double largestY = 0;
+    for (final PathCommand command in _commands) {
+      switch (command.type) {
+        case PathCommandType.move:
+          final MoveToCommand move = command as MoveToCommand;
+          smallestX = math.min(move.x, smallestX);
+          smallestY = math.min(move.y, smallestY);
+          largestX = math.max(move.x, largestX);
+          largestY = math.max(move.y, largestY);
+          break;
+        case PathCommandType.line:
+          final LineToCommand move = command as LineToCommand;
+          smallestX = math.min(move.x, smallestX);
+          smallestY = math.min(move.y, smallestY);
+          largestX = math.max(move.x, largestX);
+          largestY = math.max(move.y, largestY);
+          break;
+        case PathCommandType.cubic:
+          // TODO: Handle this case.
```

Comment:
Need to look up how skia does this

Reply:
https://bugs.chromium.org/p/skia/issues/detail?id=5751

Seems like we should just use the bounds that contain all control points

Reply:
Impeller has easier code for this too.

I'm slightly of the mind of exposing a bunch of Skia's path stuff (i.e. bounds calculation, tight bounds calculation, path operations) in a C++ lib and consuming that from this, similar to the plan for tessellation.

Reply:
depends on how complicated it is to add cubics. Currently this code is highly trivial